pinot-common/src/main/java/com/linkedin/pinot/common/metadata/segment/RealtimeSegmentZKMetadata.java

```diff
@@ -105,14 +102,39 @@ public int hashCode() {
     configMap.put(CommonConstants.Segment.Realtime.STATUS, _status.toString());
     configMap.put(CommonConstants.Segment.SEGMENT_TYPE, SegmentType.REALTIME.toString());
     configMap.put(CommonConstants.Segment.FLUSH_THRESHOLD_SIZE, Integer.toString(_sizeThresholdToFlushSegment));
+    configMap.put(CommonConstants.Segment.FLUSH_THRESHOLD_TIME, _timeThresholdToFlushSegment);
     return configMap;
   }
 
+  public Status getStatus() {
```

Comment:
nit: why move these methods?

Reply:
I wanted to have all the getters and setters together